tests/integration/expected_test_results/api_cgi_suggest/categories-string-strawberry.json

```diff
@@ -1,4 +1,5 @@
 [
+   "Mint-flavoured syrup with sugar diluted in water",
```

Comment:
This entry is because we have a weird CIQUAL category that applies to both strawberry and mint flavoured syrups... To be fixed in the taxonomy.